src/components/CurrentScenarioSupersetReport/CurrentScenarioSupersetReport.js

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+import { Backdrop, Box, CircularProgress, Stack, Typography } from '@mui/material';
+import { ErrorBoundary, SupersetEmbed } from '@cosmotech/ui';
+import { RUNNER_RUN_STATE } from '../../services/config/ApiConstants';
+import { useCurrentScenarioSupersetReport } from './CurrentScenarioSupersetReportHook';
+
+const CurrentScenarioSupersetReport = ({
+  alwaysShowReports,
```

Comment:
See below : If props are no set as required in props types, it is better to set them a fallback.